lib/cn/csched_sp3.c

```diff
@@ -1998,6 +1993,9 @@ sp3_schedule(struct sp3 *sp)
             break;
 
         case wtype_garbage:
+            if (get_time_ns() < sp->check_big_ns)
+                break;
+
```

Comment:
GC and scatter jobs now must wait a short time between the starting of new jobs.  This lets the system organically adjust to the new workload and give the throttle time to respond.  Much less jarring to the system than kicking off a bunch of heavy-media-use jobs all within the same few milliseconds.  With any luck it could help bring down some 4-9's latencies?